packages/app/src/systems/Transaction/utils/tx.ts

```diff
@@ -682,5 +691,20 @@ export function parseTx({
     isStatusPending: isStatusPending(gqlStatus),
     type,
     status,
+    time,
   };
 }
+
+export function getOperationDirection(operation: Operation, owner: string) {
+  const operationAddr = operation?.to?.address ?? operation?.from?.address;
+
+  if (!owner?.length || !operationAddr) {
+    return OperationDirection.unknown;
+  }
+
+  const ownerAddr = Address.fromString(owner);
+
+  return ownerAddr.equals(Address.fromString(operation?.to?.address ?? ''))
+    ? OperationDirection.from
+    : OperationDirection.to;
+}
```

Comment:
I think this function should live somewhere else than `tx.ts` file.

reasons:
- this code will be moved to SDK soon, and this function shouldn't be in SDK I think
- this function doesn't have unit tests (all the other methods in this file are extensively tested)